src/tools/align/components/results/AlignLabel.tsx

```diff
@@ -50,12 +66,19 @@ const AlignLabel: FC<Props> = ({ accession, info, loading, children }) => {
   }
 
   return (
-    <span className={className}>
+    <span className={className} style={style}>
+      {withCheckbox && <input type="checkbox" />}
       {reviewImg}
       {before}
       {/* inject a link to the entry page */}
       <Link to={`/uniprotkb/${accession}`}>{accession}</Link>
-      {after.join(accession)}
+      {onIdClick ? (
```

Comment:
Is it on purpose that this is only applied to the part after the extracted accession?
If no accession is detected we will just get a `before` and that's it.
Also, the whole label should react to the selection, ideally all except the link, but oh well, the link is taking you on a different page anyway, so it's fine, nobody will see the effect of the change of selected id

Reply:
As per our conversation we are going to leave this for now until UX review is done, so we are clear what needs to be done.